webapp/channels/src/i18n/tr.json

```diff
@@ -3903,9 +3957,12 @@
   "licensingPage.infoBanner.startTrialTitle": "30 gün ücretsiz deneyin!",
   "licensingPage.overageUsersBanner.cta": "Satış ekibi ile görüşün",
   "licensingPage.overageUsersBanner.ctaExpandSeats": "Ek koltuk lisansı satın alın",
+  "licensingPage.overageUsersBanner.ctaUpdateSeats": "Koltuk sayısını güncelle",
   "licensingPage.overageUsersBanner.noticeDescription": "Bir sonraki lisans sayısı artırımı denetiminizde müşteri başarısı yöneticinizi bilgilendirin. <a>Satış ekibi ile görüşün</a>",
   "licensingPage.overageUsersBanner.noticeTitle": "Çalışma alanınızın kullanıcı sayısı ücreti ödenmiş koltuk lisansı sayınızı {seats, number} {seats, plural, one {koltuk} other {koltuk}} kadar aştı",
+  "licensingPage.overageUsersBanner.selfHostedNoticeDescription": "<a>Uyumlu kalmak için</a> ek koltuk satın alın.",
```

Comment:
@esarafianou @jupenur I've seen this a couple of times, but there is no href. is this ok to merge? or could be misused in any way?

Reply:
@jupenur gentle reminder on this.